### DISCUSSION TITLE
Website/docs gone

### DISCUSSION THREAD

**Discussion by directlinkai** [keycloak/keycloak#8676] [Q&A]
Keycloak.org is 404ing right now, as are all documentation pages

**Comment by ondrej-simon** [comment]
Same over here. Does not seem to be just a local issue at @directlinkai side. :)

**Comment by stianst** [comment]
This was caused by renaming "master" branch to "main", which for some reason GH pages decided to just unpublish everything. We're working on recovering the website right now.

**Reply by ondrej-simon** [reply]
Thank you, Stian!

**Comment by stianst** [comment]
It's back up again
